### PR TITLE
add latex template

### LINKED ISSUE #1: Latex template

### PR DESCRIPTION
**Pull request by vict0rsch**
Fix #1 

I added the citations for both our previous work. I don't think that in a research context it's relevant to complexify the latex snippet with adding BCG GAMMA and comet.ml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +131,42 @@ python examples/mnist_decorator.py
...
+  journal={arXiv preprint arXiv:1910.09700},
+  year={2019}
+}
+
+@article{lottick2019nergy,
+    title={Energy Usage Reports: Environmental awareness as part of algorithmic
+  accountability},
+    author={Kadan Lottick and Silvia Susai and Sorelle A. Friedler and Jonathan P. Wilson},
+    year={2019},
+    journal={arXiv preprint arXiv:1911.08354}
```
Comment (sorelle) [inline README.md:164]: Workshop on Tackling Climate Change with Machine Learning at NeurIPS 2019

File: README.md [hunk 2]
```diff
@@ -130,6 +131,42 @@ python examples/mnist_decorator.py
...
+
+%Uncomment if you bought additional offsets:
+%XX kg CO2eq were manually offset through \href{link}{Offset Provider}.
+
+Estimations were conducted using the \href{https://github.com/mlco2/codecarbon}{CodeCarbon emissions tracker}, a joint effort from authors of \cite{lacoste2019quantifying} and \cite{lottick2019nergy}.
+
+@article{lacoste2019quantifying,
+  title={Quantifying the Carbon Emissions of Machine Learning},
+  author={Lacoste, Alexandre and Luccioni, Alexandra and Schmidt, Victor and Dandres, Thomas},
+  journal={arXiv preprint arXiv:1910.09700},
```
Comment (sorelle) [inline README.md:155]: Workshop on Tackling Climate Change with Machine Learning at NeurIPS 2019

File: README.md [hunk 3]
```diff
@@ -130,6 +131,42 @@ python examples/mnist_decorator.py
...
+# Report your emissions: LateX template
+
+We believe that an important step towards reducing carbon emissions is the generalization of emissions reporting in papers, blog posts and publications in general. Here's an example LateX snippet you might want to use:
+
+```latex
+\usepackage{hyperref}
+
+\subsection{CO2 Emission Related to Experiments}
+
+Experiments were conducted using {cloud provider} in region {region of compute}, which has a carbon efficiency of {carbon efficiency} kgCO$_2$eq/kWh. A cumulative of {hours used} hours of computation was performed on hardware of type {hardware type} (TDP of W).
```
Comment (sorelle) [inline README.md:143]: Should we make a second alternate template in the case where they didn't use cloud compute and instead used personal computers?  I've got a high-powered machine with GPU that I use for most of this stuff...

File: README.md (hunk 2 above)
Comment (vict0rsch) [reply README.md]: done thanks!

File: README.md [hunk 4]
```diff
@@ -130,6 +131,42 @@ python examples/mnist_decorator.py
...
+@article{lottick2019nergy,
+    title={Energy Usage Reports: Environmental awareness as part of algorithmic
+  accountability},
+    author={Kadan Lottick and Silvia Susai and Sorelle A. Friedler and Jonathan P. Wilson},
+    year={2019},
+    journal={arXiv preprint arXiv:1911.08354}
+}
+```
+
+To find the carbon efficiency of your cloud region, you can look into [mlco2's data](https://github.com/mlco2/impact/blob/master/data/impact.csv). If you are using a private infrastructure, [a number of resources](https://github.com/mlco2/impact/tree/master/data#mlco2s-data) can help you find the carbon efficiency of you local grid.
```
Comment (sorelle) [inline README.md:168]: I think for the private infrastructure the right link is probably:
https://github.com/mlco2/codecarbon/tree/master/codecarbon/data/private_infra/2016

And similarly, it'd make sense to use the codecarbon link for the cloud regions too:
https://github.com/mlco2/codecarbon/tree/master/codecarbon/data/cloud

File: README.md (hunk 3 above)
Comment (vict0rsch) [reply README.md:143]: I thought about that and concluded with myself that people would be free to adapt. I agree we can definitely have a "local / private infra" template, I did not want to overload the readme. @sashavor do you have an opinion?

**Review by sorelle — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (vict0rsch) [reply README.md:168]: Good points will do

File: README.md (hunk 3 above)
Comment (sashavor) [reply README.md:143]: Can we just put 'region' instead of 'region of compute'?

**Pull request merged**